src/transfers-history/adapters/web3/SpokePoolEventsQuerier.ts

```diff
@@ -116,6 +119,7 @@ export class SpokePoolEventsQuerier implements ISpokePoolContractEventsQuerier {
           retryWithLowerBatchSize = true;
         } else {
           retryWithLowerBatchSize = false;
+          console.error(error);
           throw error;
```

Comment:
log the error here for visibility because this function runs inside Promise.allSettled() and it can error silently